src/test/java/uk/gov/homeoffice/digital/sas/balancecalculator/kafka/consumer/TimeEntryConsumerTest.java

```diff
@@ -83,13 +88,13 @@ class TimeEntryConsumerTest {
     accrualList.add(accrual2);
 
     //when
-    when(balanceCalculator.calculate(timeEntryCaptor.capture())).thenReturn(accrualList);
+    when(balanceCalculator.calculate(timeEntryCaptor.capture(), kafkaActionArgumentCaptor.capture())).thenReturn(accrualList);
     timeEntryConsumer.onMessage(message);
 
     // then
     assertThat(capturedOutput.getOut()).contains(String.format(KAFKA_SUCCESSFUL_DESERIALIZATION,
         message));
-    verify(balanceCalculator).calculate(timeEntryCaptor.capture());
+    verify(balanceCalculator).calculate(timeEntryCaptor.capture(), kafkaActionArgumentCaptor.capture());
```

Comment:
Do you think we should we be checking what gets passed to `kafkaActionArgumentCaptor`?

Reply:
Good point. Maybe we can use something like `verify(balanceCalculator).calculate(timeEntryCaptor.capture(), eq(KafkaAction.CREATE));` as `KAFKA_SUCCESSFUL_DESERIALIZATION` is a string containing `'...created'` ?
@JWeston-HO what do you think?

Reply:
Added an assertion to cover the kafkaActionArgumentCaptor.

`assertThat(kafkaActionArgumentCaptor.getValue()).isEqualTo(KafkaAction.CREATE);`